build.sbt

```diff
@@ -81,6 +81,7 @@ lazy val core = project
       "org.scodec"    %% "scodec-cats"  % "1.0.0",
       "com.beachape"  %% "enumeratum"   % "1.6.1",
       "org.tpolecat"  %% "natchez-core" % "0.0.12",
+      "com.ongres.stringprep" % "saslprep" % "1.1"
```

Comment:
This is BSD 2-clause licensed and seems safe / unlikely to change or cause dependency issues for folks downstream. The main algorithm is here, which is 100+ lines of unicode manipulation: https://gitlab.com/ongresinc/stringprep/-/blob/development/saslprep/src/main/java/com/ongres/saslprep/SaslPrep.java

Reply:
:+1: seems fine